http/analysis-service/config.py

```diff
@@ -18,7 +18,8 @@
 EPHEMERAL_MAP      = { "/dev/xvdb": "ephemeral0", "/dev/xvdc": "ephemeral1" }
 SECURITY_GROUPS    = []
 INSTANCE_PROFILE   = 'telemetry-analysis-profile'
-INSTANCE_APP_TAG   = 'telemetry-analysis-worker-instance'
+INSTANCE_APP_TAG   = 'pipeline'
+INSTANCE_TYPE_TAG   = 'telemetry-analysis-worker-instance'
```

Comment:
Maybe add a comment here indicating that `INSTANCE_TYPE_TAG` is used for deciding which ec2 instances get auto-terminated after their timeout.